hsds/dset_lib.py

```diff
@@ -66,6 +67,41 @@ def getFillValue(dset_json):
     return arr
 
 
+def _get_arr_pts(arr_points, arr_index, pt, dim=0, chunk_index=None, factors=None):
+    """ recursive function to fill in chunk locations for hyperchunk selection.
+        arr_points: numpy array of shape (num_chunks, rank)
```

Comment:
Might be more clear to reword this to something like `Recursive function which fills arr_points with the coordinates to locate a hyperchunk selection's corresponding chunks in a chunk table`.

Reply:
Sounds better. Updated